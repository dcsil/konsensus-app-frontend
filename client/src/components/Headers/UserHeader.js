/*!

=========================================================
* Argon Dashboard React - v1.2.1
=========================================================

* Product Page: https://www.creative-tim.com/product/argon-dashboard-react
* Copyright 2021 Creative Tim (https://www.creative-tim.com)
* Licensed under MIT (https://github.com/creativetimofficial/argon-dashboard-react/blob/master/LICENSE.md)

* Coded by Creative Tim

=========================================================

* The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

*/

// reactstrap components
import { Container, Row, Col } from 'reactstrap';
import React from 'react';

const UserHeader = ({ firstName, image }) => {
  const background = image ? 'url(' + image + ')' : '';
  return (
    <>
      <div
        className="header pb-8 pt-5 pt-lg-8 d-flex align-items-center"
        style={{
          minHeight: '600px',
          backgroundImage: background,
          backgroundSize: 'cover',
          backgroundPosition: 'center top',
        }}
      >
        {/* Mask */}
        <span className="mask bg-gradient-default opacity-8" />
        {/* Header container */}
        <Container className="d-flex align-items-center" fluid>
          <Row>
            <Col lg="7" md="10">
              <h1 className="text-white">Hello {firstName}</h1>
              <p className="text-white mt-0 mb-5">
                This is your profile page. You can update any of your
                settings here.
              </p>
            </Col>
          </Row>
        </Container>
      </div>
    </>
  );
};

export default UserHeader;
