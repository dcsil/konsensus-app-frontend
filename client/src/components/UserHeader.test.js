import React from 'react';
import { shallow } from 'enzyme';
import UserHeader from './Headers/UserHeader';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

test('Test User Header component', () => {
  const wrapper = shallow(<UserHeader />);
  expect(wrapper).toMatchSnapshot();
});
