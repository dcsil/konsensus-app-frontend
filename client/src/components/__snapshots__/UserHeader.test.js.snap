// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test User Header component 1`] = `ShallowWrapper {}`;
